public/app/features/transformers/utils.test.ts

```diff
@@ -49,7 +49,11 @@ describe('validator', () => {
     expect(numberOrVariableValidator('$foo')).toBe(true);
   });
 
-  it('fails a string that has multiple variables', () => {
```

Comment:
I wasn't sure why we would want this to fail.. Do two or more valid variables count as invalid? I don't know why anyone would use multiple variables here, but it doesn't seem like a validity problem.

I tested to see if it would work - if I have a variable `${cutoff}` that evaluates to 34, and I put in `${cutoff}${cutoff}`, it evaluates to 3434.  Again, I don't know why anyone would do this, but it doesn't seem invalid per se. Let me know what you think.

Reply:
do some git archaeology to see when/why it was added. maybe there's a specific issue.

Reply:
i do agree that it's a weird restriction

Reply:
I did do a blame, it was added with everything else done by @oscarkilhed in https://github.com/grafana/grafana/pull/75372 to add this functionality, so it wasn't added in response to a bug or anything.

Looking at the PR, I think the test was added to respond to Kyles comment here https://github.com/grafana/grafana/pull/75372#discussion_r1341598703 - However that comment is assuming we don't support multiple variables at once when we actually do. I feel like we're safe to do this.

Reply:
:+1:  sounds good